dspace-api/src/main/java/org/dspace/util/SimpleConverter.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.util;
+
+/**
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk@4science.com)
+ */
+public interface SimpleConverter {
```

Comment:
This new interface is missing JavaDocs to describe it.  The method below should also have basic JavaDocs (even though it sounds like it might be some sort of lookup method).